app/policies/case/sar/offender_policy.rb

```diff
@@ -7,4 +7,8 @@ def transition?
   def can_add_note_to_case?
     true
   end
+
+  def can_record_data_request?
+    self.case.offender_sar?
+  end
```

Comment:
Are there ever any cases for `case/sar/offender_policy` that `self.case` could be something other than an offender_sar? Just wondering why not say "true" here

Reply:
errrr..... good point!  I think I did it to explicitly tell the reader you can only record a data request if the current case is an Offender SAR but I suppose that's a given!